pkgs/by-name/ru/rutabaga_gfx/package.nix

```diff
@@ -40,8 +40,26 @@ stdenv.mkDerivation (finalAttrs: {
     })
     # Install the dylib on Darwin.
     ./darwin-install.patch
+    # Patch for libc++, drop in next update
+    # https://issuetracker.google.com/issues/392934337
+    (fetchpatch {
+      url = "https://chromium.googlesource.com/crosvm/crosvm/+/8ae3c23b2e3899de33b973fc636909f1eb3dc98c%5E%21/?format=TEXT";
+      decode = "base64 -d";
+      hash = "sha256-RL9W2nU3bVuNu23eNOyY6Ne6Lm5xz8bBvTqvyskFSPc=";
+    })
   ];
 
+  # Adds std::env since it isn't in the current release.
+  # Drop during the next update.
+  postPatch = ''
+    echo "use std::env;" | cat - rutabaga_gfx/build.rs > rutabaga_gfx/build.rs.tmp
+    mv rutabaga_gfx/build.rs.tmp rutabaga_gfx/build.rs
+  '';
```

Comment:
At the point we're doing a fetchpatch and then manually fixing it up in postPatch, it's probably better to just commit a patch file that does the right thing.

(Sorry, I thought I included this in my review yesterday, but it looks like I either forgot to press the button, or GitHub ate it.  Probably the former.)

Reply:
I found it was easier to bump to 0.1.3 which does have the change.

Reply:
Turns out, we nee aom_logging which it seems we don't have so I just modified the patch.

Reply:
As I mentioned on Matrix the other day, 0.1.3 is [buggy](https://lore.kernel.org/qemu-devel/ufctubdq3cyltmtgwc4ng2dn6akazaybqvz5f5a6pyvgwmsxw2@3kapq5mvvy7l/).  It's on purpose that we're not on it.  We can upgrade to the next release, whenever it comes, but not 0.1.3.

Anyway, the postPatch can be deleted now, right?

Reply:
Oh, I didn't see that about 0.1.3. Yes, I removed the post patch already.

Reply:
For whatever reason, git didn't commit that change when I messed with the patch. The post patch should be gone now.